### ISSUE TITLE
Issue/edit on GitHub icons in documentation toolbar

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tskit-site#122]
I think we used to have icon(s) for "open an issue about this" and "edit on GitHub" on the documentation pages such as https://tskit.dev/tskit/docs/stable/python-api.html - I found this useful: has it been deliberately omitted? I think at least a simple on-click way to raise an issue about typos etc in the docs would help us improve them.

**Comment by jeromekelleher** [comment]
Yes, they were handy. Do they come up when you build e.g. msprime docs locally?

**Comment by hyanwong** [comment]
I can't build docs locally ATM because of that numpy issue. But the icons are there on the tutorials site:

https://tskit.dev/tutorials/tables_and_editing.html#sec-tables

**Comment by benjeffery** [comment]
Ahhh I think in fixing the duplicate buttons on some of the docs, I have removed them  on those that didn't have a duplicate. Will fix.
